ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -9,9 +11,14 @@
 REQUEST_CONTEXT_KEY = 'datadog_context'
 REQUEST_SPAN_KEY = '__datadog_request_span'
 
+PARENT_TRACE_HEADER_ID = 'x-ddtrace-parent_trace_id'
+PARENT_SPAN_HEADER_ID = 'x-ddtrace-parent_span_id'
+
+_SPAN_MIN_ERROR = 400
```

Comment:
at the moment I think we should keep this out; while I know that it's really helpful for you, I think that this PR should not include the change so we can easily proceed with the merge without waiting the specification for "what should be considered an error by default" and "how users can change that behavior in general".

Is it fine if you remove the `_SPAN_MIN_ERROR` from this PR, moving this in your personal fork (or another PR if you prefer).